test/inductor/test_auto_functionalize.py

```diff
@@ -356,6 +742,23 @@ def f(x):
         x = torch.zeros(100, dtype=torch.int64)
         f(x)
 
+    # def test_inference_mode1(self):
+    #     with torch.inference_mode():
+    #         self.test_auto_functionalize_extra1()
+
+    # def test_inference_mode2(self):
+    #     with torch.inference_mode():
+    #         self.test_auto_functionalize_extra2()
```

Comment:
can decorate with unittest.expectedFailure if these fail

Reply:
I will just add a tets that shows the results (the are not inplaced now_ and change the test once we fix it.